### ISSUE TITLE
Add CornerRadius for Controls/Views

### ISSUE THREAD

**Issue by luizproenca** [Microsoft/xaml-standard#202]
I'm porting some controls from uwp to forms and I see there is not a simple way to add rounded corners to controls like boxview, button, combobox, grid, views in general.

Using custom controls is really tiring.

**Comment by mdtauk** [comment]
If there is going to be a style/template/render engine, as part of XAML Standard - Corner Radius makes a lot of sense on all UI elements.

**Comment by insinfo** [comment]
I fully agree, it's very stressful to be creating custom controls for trivial things.

**Comment by rodrigoelp** [comment]
Especially if you want to be consistent with material/metro/whatever-microsoft-calls-it-know design

**Comment by Duranom** [comment]
It would be nice if the standard for graphical usage defines that all controls that influence layout should have it like currently in UWP Grid, RelativePanel, StackPanel, ContentPresenter (and few others controls have it as well). With controls itself I wonder if it should be done through templating or as property as you always want a bit more.

**Comment by michael-hawker** [comment]
I like @rodrigoelp's point.  I see this akin to CSS border-radius which can be applied in a multitude of ways.

**Comment by safeery2k** [comment]
Its a pain. I am doing a Xamarin Forms app and supporting all 3 platforms. My client wants rounded corners on few of the frames and they look amazing on iOS and Android but guess what UWP does not support it.

**Comment by michael-hawker** [comment]
@safeery2k what Win 10 version are you supporting? 16299 fixed support for [CornerRadius](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Grid).

**Comment by dotMorten** [comment]
@safeery2k If you're already using Xamarin.Forms you really have no need for a XAML standard. The Forms is its own cross-platform markup already.
The fact that rounded corners are ignored on one platform would be a bug in Xamarin.Forms and you should log that there.
